packages/core/gulpfile.js

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+const gulp = require('gulp');
+const serverDir = './src/common/i18n/';
+const extensionDir = 'lib/common/i18n';
+
+gulp.task('copy_json_files', () => {
+    gulp.src(serverDir + '*.json')
+        .pipe(gulp.dest(extensionDir))
+});
+
+
+
+
+
+
```

Comment:
Btw I think we could use shx cp rather than the gulp file...

See how they do in in examples/browser/package.json : build:backend script
"build:backend": "shx cp src-gen/frontend/index.html lib",